libs/cluster/Server/Migration/MigrateSessionKeys.cs

```diff
@@ -75,7 +75,7 @@ private bool MigrateKeysFromMainStore(ref List<(long, long)> keysWithSize, out L
                     if (!o.IsSpanByte)
                     {
                         memoryHandle = o.Memory.Memory.Pin();
-                        value = SpanByte.FromPinnedMemory(o.Memory.Memory);
+                        value = SpanByte.FromPinnedSpan(o.Memory.Memory.Span);
```

Comment:
I think this makes the Code more complex to understand, before it was clear what SpanByte  method to use. ``FromPinnedMemory`` -> you need to pin the memory with ``Pin()``. Now this relation is missing. There is no ``Pin()`` for span

Reply:
Sure. Reverting.